rpc/types_broadcast_transaction.go

```diff
@@ -56,6 +57,26 @@ type BroadcastDeclareV2Txn struct {
 	ContractClass     ContractClass `json:"contract_class"`
 }
 
+type BroadcastDeclareTxnV3 struct {
+	Type              TransactionType       `json:"type"`
+	SenderAddress     *felt.Felt            `json:"sender_address"`
+	CompiledClassHash *felt.Felt            `json:"compiled_class_hash"`
+	Version           NumAsHex              `json:"version"`
+	Signature         []*felt.Felt          `json:"signature"`
+	Nonce             *felt.Felt            `json:"nonce"`
+	ContractClass     *ContractClass        `json:"contract_class"`
+	ResourceBounds    ResourceBoundsMapping `json:"resource_bounds"`
+	Tip               *felt.Felt            `json:"tip"`
+	// The data needed to allow the paymaster to pay for the transaction in native tokens
+	PayMasterData []*felt.Felt `json:"paymaster_data"`
+	// The data needed to deploy the account contract from which this tx will be initiated
+	AccountDeploymentData *felt.Felt `json:"account_deployment_data"`
+	// The storage domain of the account's nonce (an account has a nonce per DA mode)
+	NonceDataMode DAMode `json:"nonce_data_availability_mode"`
+	// The storage domain of the account's balance from which fee will be charged
+	FeeMode DAMode `json:"fee_data_availability_mode"`
```

Comment:
nit: probably better to name these similarly.

Reply:
Updated to match Junos name